names

```diff
@@ -0,0 +1,20 @@
+equals
+lte
+compose
+id
+concat
+empty
+map
+contramap
+ap
+of
+alt
+zero
+reduce
+traverse
+chain
+chainRec
+extend
+extract
+bimap
+promap
```

Comment:
To export another string one will update this file, run `make`, then commit the changes to all three files (__names__, __index.js__, and __index.d.ts__).